tests/Unit/Controllers/SoundingBoardControllerTest.php

```
@@ -48,22 +52,35 @@ protected function setUp(): void
     // HELPERS
     // ===========================
 
-    private function ctrl(?FakeRequest $r = null): SoundingBoardController
-    {
+    private function ctrl(
+        ?FakeRequest $r = null,
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $r. Configured minimum length is 3. (undefined)

(ShortVariable)